tf2, css: memalloc: call to IsPowerOfTwo is ambiguous

**To reproduce:**
1. Clone and build `metamod-source`
1. Clone the `tf2` or `css` branch
1. Try building `metamod-source/sample_mm` with the cloned sdk

**Expected:**
Builds without error.

**Actual:**
`tf2` branch build crashes with error: (https://gist.github.com/maximsmol/cdda5865b2d1284029bcc85f06c0fcf8)
```
hl2sdk-tf2/public/tier0/memalloc.h:251:8: error: call to 'IsPowerOfTwo' is ambiguous
        if ( !IsPowerOfTwo( align ) )
```

`css` branch build crashes with error:
```
/hl2sdk-css/public/tier0/memalloc.h:181:7: error: call to 'IsPowerOfTwo' is ambiguous
        if (!IsPowerOfTwo(align))
```

**Potential fix:**
Given that
```
inline bool ValueIsPowerOfTwo( size_t value ) // don't clash with mathlib definition
```
already exists, it is safe to assume that it was meant to be used in all the cases where `IsPowerOfTwo` is used with a `size_t` argument to avoid the clash.

**It seems like this issue might be common to all the SDK versions!**

--- a/public/tier0/memalloc.h
+++ b/public/tier0/memalloc.h
@@ -178,7 +178,7 @@ inline void *MemAlloc_AllocAligned( size_t size, size_t align )
 {
 	unsigned char *pAlloc, *pResult;
 
-	if (!IsPowerOfTwo(align))
+	if (!ValueIsPowerOfTwo(align))
 		return NULL;
 
 	align = (align > sizeof(void *) ? align : sizeof(void *)) - 1;
@@ -196,7 +196,7 @@ inline void *MemAlloc_AllocAligned( size_t size, size_t align, const char *pszFi
 {
 	unsigned char *pAlloc, *pResult;
 
-	if (!IsPowerOfTwo(align))
+	if (!ValueIsPowerOfTwo(align))
 		return NULL;
 
 	align = (align > sizeof(void *) ? align : sizeof(void *)) - 1;
@@ -248,7 +248,7 @@ inline void *MemAlloc_AllocAlignedFileLine( size_t size, size_t align, const cha
 
 inline void *MemAlloc_ReallocAligned( void *ptr, size_t size, size_t align )
 {
-	if ( !IsPowerOfTwo( align ) )
+	if ( !ValueIsPowerOfTwo( align ) )
 		return NULL;
 
 	// Don't change alignment between allocation + reallocation.

